http/src/main/java/io/micronaut/http/filter/HttpFilterResolver.java

```diff
@@ -96,18 +101,21 @@ default boolean hasPatterns() {
          * @param filter The filter
          * @param annotationMetadata The annotation metadata
          * @param methods The methods
+         * @param patternStyle the pattern style
          * @param patterns The patterns
          * @return The filter entry
          * @param <FT> the filter type
          */
         static <FT extends HttpFilter> FilterEntry<FT> of(
```

Comment:
Need to create a new method here instead of changing the existing one

Reply:
done